pkg/os/darwin/release_info.go

```diff
@@ -20,8 +21,9 @@ var (
 
 func AtLeast(targetVersion string) (bool, error) {
 	once.Do(func() {
-		macCurrentVersion, errGettingVersion = syscall.Sysctl("kern.osproductversion")
-		logging.Debugf("kern.osproductversion is: %s", macCurrentVersion)
+		macCurrentVersion, _, errGettingVersion = os.RunWithDefaultLocale("sw_vers", "-productVersion")
+		macCurrentVersion = strings.TrimTrailingEOL(macCurrentVersion)
+		logging.Debugf("sw_vers --productVersion is: %s", macCurrentVersion)
```

Comment:
This needs a comment explaining why we cannot use `syscall.Sysctl("kern.osproductversion")` directly / a link to the go issue.

Reply:
mentioned in the commit log, but sure can add comment here.